doc/sphinx/proof-engine/ltac.rst

```diff
@@ -174,6 +174,9 @@ mode but it can also be used in toplevel definitions as shown below.
    ltac_def         : `ident` [`ident` ... `ident`] := `ltac_expr`
                     : `qualid` [`ident` ... `ident`] ::= `ltac_expr`
 
+
+.. todo: document "@selector: "
```

Comment:
I should have said "command selectors", which I think is not documented.  The reference is for tactics.  Where should we document command selectors?

Reply:
There is no such thing as a tactic or a command selector. These are goal selectors. And they are the same whether applied to a tactic or a command. We should gather all commands that support goal selectors in a single section and document it here.

Reply:
There are selectors applied to tactics and selectors applied to commands (`query_commands` such as `Eval`, `Compute`, `Search` commands and `Info`).  The `Info` command also accepts `par :` which looks like a selector.

The tactics chapter describes some of this in the "Invocation of tactics" section, while the Ltac chapter has a better "Goal selectors" section covering the same thing.  Neither one mentions using selectors in commands.  I can tweak the Ltac section and have the commands hyperlink to that section.  Also remove the details from "invocation of tactics"--or do you want to repeat them?  What do you think?

Reply:
Indeed, the documentation for this is currently all over the place. The natural thing would be that goal selectors are described in the "Proof using" chapter (currently `@num: {` and `[@ident]: {` are already described there). But in truth, these goal selectors are currently only available in "Classic" proof mode (i.e. in Ltac1 mode). I'll see if this can easily be changed.

Reply:
I will leave a todo in the tactics chapter to sort out selectors and not make any changes right now.  That seems beyond the scope of this PR.